tools/run_tests/run_interop_tests.py

```diff
@@ -1029,5 +1096,8 @@ def aggregate_http2_results(stdout):
   dockerjob.finish_jobs([j for j in server_jobs.itervalues()])
 
   for image in docker_images.itervalues():
-    print('Removing docker image %s' % image)
-    dockerjob.remove_image(image)
+    if not args.manual_run:
+      print('Removing docker image %s' % image)
+      dockerjob.remove_image(image)
+    else:
+      print('Preserving docker image: %s' % image)
```

Comment:
Yes, the build will happen every time. 
And no, it will build a "different" image each time. It takes the base image (same each time unless the base dockerfile changes) and construct the image using "docker run" and then storing the resulting image.

@yongni  yes, we could save the list of images kept, fine to do that in a followup PR? It's a tiny change.

Btw, we can fine-tune the behavior of "--manual_run" in a followup PR because it's not really used by and automated tests scripts at this point, so changing the semantics a little bit is fine.